curp/src/server/raw_curp/log.rs

```diff
@@ -252,7 +292,14 @@ impl<C: 'static + Command> Log<C> {
             {
                 continue;
             }
-
+            for e in self.entries.range(pi..) {
+                if matches!(e.entry_data, EntryData::ConfChange(_)) {
+                    let _ig = need_fallback_indexes.insert(e.index);
```

Comment:
Yes, only first truncation will enter this loop. the latter `self.entries.truncate(pi)` has the same problem. I think the better way is refactor the `try_append_entries`, just find the truncate index and truncate once, I will create a ISSUE to track this, It will not be fixed in this PR